_includes/install-info.html

```diff
@@ -0,0 +1,150 @@
+<h2>
+  <a
+    href="#"
+    class="collapsed"
+    data-toggle="collapse"
+    data-target="#installing-python-and-numpy"
+  >
```

Comment:
The icon for collapsing and expanding aren't showing for me.

Reply:
Hm, it's supposed to be using a fontawesome icon, which is included in the `default.html` layout, do you see a `use.fontawesome.com` stylesheet link in the page's `<head>`?

We could switch to doing the caret using CSS, it looks like the dropdowns in the header (e.g. after "Documentation") are drawn with CSS, maybe we can just reuse that class instead.

Reply:
Looking at this more, should we just remove the collapse? The tabbed info isn't that tall and people can just scroll by if they want?

Reply:
Yeah it's definitely not as critical having put the install stuff into tabs... I do still kind of like having this collapsed, but I'm fine either way. @drasmuss any preference?

Reply:
I see the font-awesome but still just a box. Not sure what is going on. I don't have strong feelings on removing the collapse. Likely just a build issue, so I am good either way.

Reply:
Interesting. We'll see what Dan says about the collapsing and if we still want it collapsed I'll switch it to using CSS to draw the caret.